working/templates/srmfunction/Functions/Functions.xml

```diff
@@ -0,0 +1,8 @@
+﻿<Functions xmlns="http://www.skyline.be/config/functions">
+	<Version>1.0.0.1</Version>
+	<Protocol>
+		<Name>$PROTOCOLNAME$</Name>
+	</Protocol>
+	<Function id="$FUNCTIONID$" name="$FUNCTIONNAME$" maxInstances="0" profile="$PROFILEDEFINITIONID$">
+	</Function>
```

Comment:
S: Include some examples for parameters, interfaces, graphical tags. Can be commented out.

Reply:
The function file only needs the Id and profile Id to allow the user to make use of the function editor in DIS.
The idea is to use DIS to extend the function, so no need to insert examples...